### PR TITLE
Hotjar privacy update

### PR DESCRIPTION
**Pull request by mcarlson**

### CODE REVIEW

**Review by amydarling — COMMENTED** [review]
Approved.

**Comment by Ainali** [comment]
It looks from [Hotjar's GDPR page](https://www.hotjar.com/legal/compliance/gdpr-commitment/) that we should link to their Privacy page too. (This we might even need to do from every site where we use Hotjar.)
And as far as I can understand, we should also sign a Data Processing Agreement with Hotjar.

File: organization/privacy.md [hunk 1]
```diff
@@ -8,11 +8,12 @@ redirect_from:
 
 # Privacy
 
-The Foundation strongly supports individuals' right to privacy. For this reason we don't track individual users on any of our websites but count traffic for analytics.
+The Foundation strongly supports individuals' right to privacy.
```
Comment (Ainali) [inline organization/privacy.md:11]: I think this needs to be further explained. If we really "strongly support individuals' right to privacy", we shouldn't track individual users. Removing this sentence without changing the first sentence seems contradictory, unless we have an explanation somewhere to why we feel tracking individuals is supporting that right.

Comment (mcarlson) [reply organization/privacy.md]: @Ainali this should be resolved now.

**Comment by mcarlson** [comment]
No need for this as we drew the line at adding cookie popups...

**Pull request closed without merging**